### PR TITLE
Add `Arc::flipped`

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

**Review by DJMcNab — COMMENTED** [review]
The implementation looks reasonable; what are the expected use-cases for this?

File: src/arc.rs [hunk 1]
```diff
@@ -206,3 +221,24 @@ impl Mul<Arc> for Affine {
...
+        // Most fields should be unchanged:
+        assert_eq!(a.center, f.center);
+        assert_eq!(a.radii, f.radii);
+        assert_eq!(a.x_rotation, f.x_rotation);
+
+        // Sweep angle should be in reverse
+        assert_eq!(a.sweep_angle, -f.sweep_angle);
+
+        // Flipping it again should result in the original arc
+        assert_eq!(a, f.flipped());
```
Comment (derekdreery) [inline src/arc.rs:242]: No fuzzy testing needed, since there is only addition/subtraction meaning perfect accuracy, at least for normal input.

**Review by derekdreery — APPROVED** [review]
LGTM.

@DJMcNab these kinds of operations can be useful sometimes, for applications other than drawing (when it really doesn't matter what direction the line goes in).

File: src/arc.rs (hunk 1 above)
Comment (waywardmonkeys) [reply src/arc.rs:242]: You are confirming what I have here is fine, not suggesting a change, right?

**Comment by DJMcNab** [comment]
What are the next steps here?

**Comment by waywardmonkeys** [comment]
I am going to land it before the meeting.

**Pull request merged**